packages/utils/test/envelope.test.ts

```diff
@@ -1,16 +1,82 @@
-import type { Event, EventEnvelope } from '@sentry/types';
+import type { Event, EventEnvelope, SpanAttributes } from '@sentry/types';
 
+import {
+  SEMANTIC_ATTRIBUTE_EXCLUSIVE_TIME,
+  SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_UNIT,
+  SEMANTIC_ATTRIBUTE_SENTRY_MEASUREMENT_VALUE,
+  spanToJSON,
+} from '@sentry/core';
+import { SentrySpan } from '@sentry/core';
 import {
   addItemToEnvelope,
   createEnvelope,
+  createSpanEnvelopeItem,
   forEachEnvelopeItem,
   parseEnvelope,
   serializeEnvelope,
 } from '../src/envelope';
+import { dropUndefinedKeys } from '../src/object';
 import type { InternalGlobal } from '../src/worldwide';
 import { GLOBAL_OBJ } from '../src/worldwide';
 
 describe('envelope', () => {
+  describe('createSpanEnvelope()', () => {
+    it('span-envelope-item of INP event has the correct object structure', () => {
+      const attributes: SpanAttributes = dropUndefinedKeys({
```

Comment:
I think we don't need `dropUndefinedKeys` here as we are manually building this object!